systemvm/debian/opt/cloud/bin/configure.py

```diff
@@ -908,6 +908,7 @@ def __createfile(self, ip, folder, file, data):
             if os.path.exists(metamanifest):
                 fh = open(metamanifest, "a+")
                 self.__exflock(fh)
+                fh.seek(0)
```

Comment:
@GutoVeronezi , I think your intention is
```suggestion
                fh = open(metamanifest, “w+")
                self.__exflock(fh)
```
opeing with a+ will not truncate the previous version, but only overwrite data afaik.